src/main/java/org/eclipse/cargotracker/interfaces/booking/web/CargoAdmin.java

```diff
@@ -39,8 +44,13 @@
 	private String destinationUnlocode;
 	private String trackingId;
 	private List<Leg> legs;
+	private CargoTracking cargoTrack;
 	@Inject
 	private BookingServiceFacade bookingServiceFacade;
+	@Inject
+	private CargoRepository cargoRepository;
```

Comment:
We should try to avoid direct access to the repositories and do any domain object lookup work in the facade.

Reply:
Right, I think is just something forgotten there from previous refactoring.